compliance/mlperf_compliance/constants.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 MLBenchmark Group. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Master list of constants in MLPerf log
+"""
+
+# NOTE: Keep string values in alphabetical order under each section.
+
+# Constant values - benchmark name
+GNMT = "gnmt"
+MASKRCNN = "maskrcnn"
+MINIGO = "minigo"
+NCF = "ncf"
+RESNET = "resnet"
+SSD = "ssd"
+TRANSFORMER = "transformer"
+
+# Constant values - model info
+ADAM = "adam"
+LARS = "lars"
+LAZY_ADAM = "lazy_adam"
+SGD = "sgd"
+
+# Log keys - submission info
+SUBMISSION_BENCHMARK = "submission_benchmark"
+SUBMISSION_DIVISION = "submission_division"
+SUBMISSION_ENTRY = "submission_entry"
+SUBMISSION_ORG = "submission_org"
+SUBMISSION_PLATFORM = "submission_platform"
+SUBMISSION_POC_NAME = "submission_poc_name"
+SUBMISSION_POC_EMAIL = "submission_poc_email"
+SUBMISSION_STATUS = "submission_status"
+
+# Log keys - timing info
+BLOCK_START = "block_start"
+BLOCK_STOP = "block_stop"
+EPOCH_START = "epoch_start"
+EPOCH_STOP = "epoch_stop"
+EVAL_START = "eval_start"
+EVAL_STOP = "eval_stop"
+INIT_START = "init_start"
+INIT_STOP = "init_stop"
```

Comment:
It is a bit confusing to have 1)RUN_START and INIT_START/INIT_STOP here in constants.py  but only  RUN_START/RUN_STOP tag in tags.py. 
Is constants.py designed to fully deprecate tags.py? 
If so, the v0.5 way of calling print(key=tags) would not easily work for new constants/tags under v0.6 requirement. 
Not a big deal, but just cost a little burden to make logging compatible with v0.6...

Reply:
Also, coincidently, last round in v0.5 Google used tf_cnn_benchmark repo, where it contains constants.py also. So this mlperf constants.py will collide with the constants.py file there. Suggesting renaming to mlperf_constants.py

Reply:
Ok, no need to rename. "from mlperf_compliance import constants as mlperf_constants" my bad.

Reply:
the constants.py is supposed to replace the tags.py, the reason I did not edit tags.py directly is partly because the reference implementations are still using v0.5 logger and are not going to be updated immediately, also partly because I am trying to simplify things a bit by putting all constants together vs. having separate model tags files.
If you think it would be more convenient (for backward compatibility) to have the tags.py include a union of both v0.5 and v0.6 tags, then I can add that too.
Please also be reminded that with this patch you might want to use the new mlperf_print() call to print the logs, so as to adapt to the new logging format. This was also introduced in an attempt to simplify the API by using a single function vs. separate functions for each models.

Reply:
Thanks, I think it is best to merge your PR as is. I have been able to find workarounds as I mentioned.